NitroxServer/Program.cs

```diff
@@ -57,6 +58,51 @@ static void configureCultureInfo()
 
             Thread.CurrentThread.CurrentCulture = cultureInfo;
             Thread.CurrentThread.CurrentUICulture = cultureInfo;
+        }
+
+        private static void CatchExitEvent()
+        {
+            // Catch Exit Event
+            PlatformID platid = Environment.OSVersion.Platform;
+
+            // using *nix signal system to catch Ctrl+C
+            if (platid == PlatformID.Unix || platid == PlatformID.MacOSX || platid == PlatformID.Win32NT || (int)platid == 128) // mono = 128
+            {
+                Console.CancelKeyPress += new ConsoleCancelEventHandler(OnCtrlCPressed);
+            }
+
+            // better catch using WinAPI. This will handled process kill
+            if (platid == PlatformID.Win32NT)
+            {
+                consoleHandler = new ConsoleEventDelegate(ConsoleEventCallback);
+                SetConsoleCtrlHandler(consoleHandler, true);

```

Comment:
Can you remove the `consoleHandler` var here and from the class itself? Just set the SetConsoleCtrlHandler directly.